source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+
+The **port-security** attribute is set at port creation time, and its value will
+be inherited from the corresponding *network* entity when not defined at port
+level. Afterwards, it can be updated through the API - at port *or* network
+levels.
+
+The default behavior for the network's **port-security** attribute would be
+defined in the configuration file, through the ***port-security-enabled***,
+property, located in a new section, called **NETWORK**. This value will be set
+during *engine-setup* - refer to [this section](#engine-setup) - and the
+default choice will be 'yes', which will result in all traffic being dropped to
+the VMs.
+
+Existent ports can later be updated, disabling the **port-security** attribute,
+which would remove the ACLs that drop all IP traffic to the VM.
+
+As defined in the [network port security attribute definition](https://developer.openstack.org/api-ref/network/v2/#port-security), updating the **port-security** attribute on a network object **does not** cascade the value to ports attached to that network - meaning that effect would only apply to ports created *after* that update. Check the [upgrade](#installation/upgrade) section for the consequences of this requirement on ovirt-provider-ovn upgrades.
+
+## User workflow
+
+Since the feature is active by default, the intended workflow - from the oVirt
+engine user's perspective - is:
+
+* create an external network on the ovirt-provider-ovn - its default
+*port_security_enabled* value is true - which can be overriden during engine-setup.
+* create VMs attached to this external network - they will inherit the
+*port_security_attribute* from the network, which will result in dropping all
+IP traffic to that VM.
+* the intended traffic will have to be white-listed. Assuming the user is
+interested in allowing ssh traffic, the following ansible playbook should be
+used, which creates one security group - for which a security group rule
+allowing all **egress** traffic is automatically created -, plus one security
+group rule allowing **ingress** tcp traffic meant for port 22:
+
+~~~~~~
+
+- os_security_group:
+    cloud: ovirt
+    state: present
+    name: my_app_default
+    description: allow ssh
+    register: my_app_default_security_group
+
+- os_security_group_rule:
+    cloud: ovirt
+    state: present
+    security_group: "{{ my_app_default_security_group.id }}"
+    direction: ingress
+    protocol: tcp
+    port_range_min: 22
+    port_range_max: 22
+
+~~~~~~
+
+* finally, the user is required to update the ports, indicating which security
+groups apply to it. The following playbook shows how a single port, referenced
+by name, is updated:
+
+~~~~~~
+
+- os_port:
+    cloud: ovirt
+    state: present
+    name: <port_name>
+    security_groups:
+        - "{{ my_app_default_security_group.id }}"
+
+~~~~~~
+
+### Installation/Upgrade
+
+The install time required updates are described in the section [above](#engine-setup).
```

Comment:
"above" scrolls down and not up.. Not critical but please notice

Reply:
Nice spotting this one; I have moved the section to a different place and didn't realize that.

Done